zerver/views/video_calls.py

```diff
@@ -207,9 +207,9 @@ def get_bigbluebutton_url(request: HttpRequest, user_profile: UserProfile) -> Ht
 @has_request_variables
 def join_bigbluebutton(
     request: HttpRequest,
-    meeting_id: str = REQ(json_validator=check_string),
-    password: str = REQ(json_validator=check_string),
-    checksum: str = REQ(json_validator=check_string),
+    meeting_id: str = REQ(),
+    password: str = REQ(),
+    checksum: str = REQ(),
 ) -> HttpResponse:
```

Comment:
This surely requires changes to the JS code in `compose.js` that uses this endpoint.